lib/compilers/dotnet.ts

```diff
@@ -900,10 +904,14 @@ do()
         // prettier-ignore
```

Comment:
Can you search for prettier-ignore and remove/replace with the equivalent for biome? Folks have deliberately disbaled the formatter for some areas of code where it's nice to convey meaning here.

Reply:
Yep good call